### ISSUE TITLE
Don't lose the stroke of the geometry tool when moving, rotating or zooming the canvas

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#3949]
**Is your feature request related to a problem? Please describe**

If you draw a shape and move, rotate or zoom the canvas, the path disappears.

https://user-images.githubusercontent.com/74098734/117965633-e9533c80-b31a-11eb-9b8b-d67d32f2ad9a.mp4

**Describe the solution you'd like**

Hold the path when you move around the canvas.

https://user-images.githubusercontent.com/74098734/117966246-9928aa00-b31b-11eb-90bc-64c1348fecf2.mp4

Interestingly, the stroke remains if you move by pressing the mouse wheel.

**Comment by beeheemooth** [comment]
This feature is already implemented in Tahoma 2D.

**Comment by F-Burning** [comment]
Yes, I'm using Tahoma in the second video, but I would like to see it in Opentoonz.

**Comment by RodneyBaker** [comment]
Adding 'roadmap' label as there is working code available in Tahoma2D that can be ported to add this capability.
